wdtk-dumpfiles/src/main/java/org/wikidata/wdtk/dumpfiles/DumpProcessingController.java

```diff
@@ -374,7 +374,8 @@ public Sites getSitesInformation() throws IOException {
 	 * @see DumpProcessingController#processDump(MwDumpFile)
 	 * @see DumpProcessingController#getMostRecentDump(DumpContentType)
 	 */
-	public void processAllRecentRevisionDumps() {
+	public void processAllRecentRevisionDumps() 
+		throws IOException, FileAlreadyExistsException {
```

Comment:
`FileAlreadyExistsException` is a sub class of `IOException` (same in other places)

Reply:
Perhaps there is still a case for including them explicitly in the signature, because the caller might want to handle them differently?